utensor_cgen/legalizer/onnx.py

```diff
@@ -0,0 +1,57 @@
+from utensor_cgen.ir import OperationInfo, TensorInfo
+from utensor_cgen.utils import topologic_order_graph
+
+from .base import LegalizerBase
+
+
+class OnnxLegalizer(LegalizerBase):
+  TARGET = 'onnx'
+
+  def legalize(self, ugraph):
+    self._visit(ugraph)
```

Comment:
A bit weird use of name visit, normally `visit` calls are on individual nodes, and not the whole graph. Consider using a `visit_all` method or similar

Reply:
Yes, that's right.
Will rename it as `visit_all`